1209/D706/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: pid
+title: SkyDrop
+owner: SkyBean
```

Comment:
The capitalisation of the 'owner' line must match the directory name you used ('skybean'). Other than that, this PR looks fine.